userprofile/views.py

```diff
@@ -221,3 +224,71 @@ def get_context_data(self, **kwargs):
         )
 
         return context
+
+class TeacherListView(UserProfileView):
+    access_mode = ACCESS.SUPERUSER
+    template_name = "userprofile/teachers.html"
+
+    def entry(self, teacher: UserProfile, instance: CourseInstance):
+        return [
+            teacher.user.first_name + " " + teacher.user.last_name,
+			teacher.user.email,
+			str(instance),
+			str(instance.starting_time),
+			str(instance.ending_time),
```

Comment:
What are these extra indents? (Does the indentation of this array use both spaces and tabs?)